lib/OpenQA/Test/TimeLimit.pm

```diff
@@ -1,25 +1,25 @@
-# Copyright 2020-2021 SUSE LLC
+# Copyright SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 package OpenQA::Test::TimeLimit;
 use Test::Most;
+use experimental 'signatures';
 
 my $SCALE_FACTOR = $ENV{OPENQA_TEST_TIMEOUT_SCALE_FACTOR} // 1;
 
-sub import {
-    my ($package, $limit) = @_;
+sub import ($package, $limit = undef) {
     die "$package: Need argument on import, e.g. use: use OpenQA::Test::TimeLimit '42';" unless $limit;
     # disable timeout if requested by ENV variable or running within debugger
     return if ($ENV{OPENQA_TEST_TIMEOUT_DISABLE} or $INC{'perl5db.pl'});
     $SCALE_FACTOR *= $ENV{OPENQA_TEST_TIMEOUT_SCALE_COVER} // 3 if Devel::Cover->can('report');
-    $SCALE_FACTOR *= $ENV{OPENQA_TEST_TIMEOUT_SCALE_CI}    // 2 if $ENV{CI};
-    $limit        *= $SCALE_FACTOR;
+    $SCALE_FACTOR *= $ENV{OPENQA_TEST_TIMEOUT_SCALE_CI} // 2 if $ENV{CI};
+    $limit *= $SCALE_FACTOR;
     $SIG{ALRM} = sub { BAIL_OUT "test '$0' exceeds runtime limit of '$limit' seconds\n" };
     alarm $limit;
 }
 
-sub scale_timeout {
-    return $_[0] * $SCALE_FACTOR;
+sub scale_timeout ($time) {
+    return $time * $SCALE_FACTOR;
```

Comment:
Please use separate commits for style changes in existing files so that the focus of the most important commit becomes clear